numpy/random/mtrand/mtrand.pyx

```diff
@@ -551,6 +552,17 @@ cdef object discd_array(rk_state *state, rk_discd func, object size, ndarray oa,
                 PyArray_MultiIter_NEXTi(multi, 1)
     return array
 
+cdef inline void compute_complex(double *rv_r, double *rv_i, double loc_r,
```

Comment:
`compute_complex` is a very generic name that doesn't say anything about the function.

Reply:
Well, there _are_ two hard problems in computer science....